### PR TITLE
Payara 1594 jca support in payara micro

### PR DESCRIPTION
**Pull request by mikecroft**

### CODE REVIEW

**Comment by mikecroft** [comment]
Tried to keep this one solely on JCA and deploying a RAR file. Thought it was also helpful to mention the deploy order.

I'll be adding the full worked example as a user guide - would it be useful to link to it in a "See Also" section as a footer to this page?

**Comment by OndrejM** [comment]
> would it be useful to link to it in a "See Also" section as a footer to this page?
Why not, links to resources are always useful and it helps keep the documentation concise

**Comment by mikecroft** [comment]
Latest build with user guide @OndrejM 
https://mikecroft.gitbooks.io/payara-server-documentation-fork/content/v/ac7466d99105eb64fdd3475d01f2403b83388dcf/documentation/user-guides/mdb-in-payara-micro.html

File: documentation/payara-micro/jca.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+available to all types of EJB.
+
+=== Installing a JCA Adapter
+
+A JCA Resource Archive (RAR) is installed by deploying it to Payara Micro in 
+the same way as an application archive (e.g. a WAR) is deployed. The following
+shows an example using an ActiveMQ (JMS 1.1) RAR:
+
+----
+java -jar payara-micro.jar --autobindhttp --deploy activemq-rar-5.14.5.rar
```
Comment (OndrejM) [inline documentation/payara-micro/jca.adoc]: I would remove the `autobind` argument as it's not necessary, or is it?

File: documentation/payara-micro/jca.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+----
+
+The connector is subsequently available to all other applications deployed to
+Payara Micro. An application using a connector and the connector itself can be
+deployed simultaneously:
+
+----
+java -jar payara-micro.jar --autobindhttp --deploy activemq-rar-5.14.5.rar --deploy myapp.war
```
Comment (OndrejM) [inline documentation/payara-micro/jca.adoc]: Again, remove `autobind`

File: documentation/user-guides/mdb-in-payara-micro.adoc [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+ of the ActiveMQ installation as much as possible. Following the above example,
+the driver is activemq-rar-5.14.5.rar and can be downloaded from:
+https://mvnrepository.com/artifact/org.apache.activemq/activemq-rar/5.14.5
+
+=== Installing the driver
+A JCA rar is installed by deploying it to Payara Micro in the same way as an
+application archive (e.g. a .war) is deployed. The following shows an example:
+
+----
+java -jar payara-micro.jar --autobindhttp --deploy activemq-rar-5.14.5.rar
```
Comment (OndrejM) [inline documentation/user-guides/mdb-in-payara-micro.adoc]: Again the autobind argument

File: documentation/user-guides/mdb-in-payara-micro.adoc [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+----
+
+The connector is subsequently available to all other applications deployed to
+Payara Micro. An application using a connector and the connector itself can be
+deployed simultaneously:
+
+----
+java -jar payara-micro.jar --autobindhttp --deploy activemq-rar-5.14.5.rar --deploy myapp.war
```
Comment (OndrejM) [inline documentation/user-guides/mdb-in-payara-micro.adoc]: Again the autobind argument

File: documentation/user-guides/mdb-in-payara-micro.adoc [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+----
+
+=== Sending messages
+Sending messages to ActiveMQ can be done via the JMS API. At the moment, 
+ActiveMQ 5.x only supports the _JMS 1.1_ API, so even though Payara Micro supports
+JMS 2.0, this can not be used in this example. Attempting to use JMS 2.0 anyway
+would result in exceptions like show below:
```
Comment (OndrejM) [inline documentation/user-guides/mdb-in-payara-micro.adoc]: "show**n**" instead of "show"

Comment (arjantijms) [inline documentation/user-guides/mdb-in-payara-micro.adoc]: Good one, missed it. Thx!

File: documentation/payara-micro/jca.adoc (hunk 1 above)
Comment (arjantijms) [inline documentation/payara-micro/jca.adoc]: Doesn't seem to indeed. I just used it because it was in Steve's examples on which the text is based.

**Pull request merged**